charts/sgx-device-plugin/templates/sgx.yaml

```diff
@@ -7,6 +7,7 @@ apiVersion: deviceplugin.intel.com/v1
 kind: SgxDevicePlugin
 metadata:
   name: {{ .Values.sgx.metadata.name }}
+  annotations: {{ toYaml .Values.sgx.metadata.annotations | nindent 4 }}
```

Comment:
don't we have to loop `range` here?

Reply:
It's not necessary. _.annotations_ is converted into an object by toYaml which is then indented appropriately.

Reply:
thanks